### PR TITLE
Use persp-set-keymap-prefix instead of setq

### PR DESCRIPTION
**Pull request by shackra**
this is to change the default keymap of persp-mode correctly and avoid
weird bugs

- https://github.com/Bad-ptr/persp-mode.el/blob/298df111f081b5925f0aa0126a1b8d334117e0a2/persp-mode.el#L1002-L1017
- https://www.reddit.com/r/DoomEmacs/comments/qgn814/adding_a_specific_package_as_a_module/?utm_source=share&utm_medium=web2x&context=3
- https://github.com/merrickluo/lsp-tailwindcss/issues/22

Under [some circumstances](https://www.reddit.com/r/DoomEmacs/comments/qgn814/adding_a_specific_package_as_a_module/?utm_source=share&utm_medium=web2x&context=3) `:config (default +bindings)` will fail to rebind the keymap of persp-mode which in turn breaks some bindings for projectile, to prevent this bug from happening, using a function [provided by persp-mode](https://github.com/Bad-ptr/persp-mode.el/blob/298df111f081b5925f0aa0126a1b8d334117e0a2/persp-mode.el#L1002-L1017) to change the keymap is required.

Using [`setq` to change a `defcustom`](https://emacs.stackexchange.com/questions/44220/how-to-overwrite-a-defcustom-of-a-package-within-the-initfile#comment68327_44223) shouldn't be done anyway.

### CODE REVIEW

**Comment by shackra** [comment]
I know this is a small change, but I wonder if something is missing that could get this approved and merge

File: modules/config/default/+emacs-bindings.el [hunk 1]
```diff
@@ -5,7 +5,7 @@
       doom-localleader-alt-key "C-c l")
 
 ;; persp-mode and projectile in different prefixes
-(setq persp-keymap-prefix (kbd "C-c w"))
+(persp-set-keymap-prefix (kbd "C-c w"))
```
Comment (hlissner) [inline modules/config/default/+emacs-bindings.el:8]: Can you try `(setq! persp-keymap-prefix (kbd "C-c w"))` instead?

I'd rather avoid having to eagerly load the package just for this.

**Review by hlissner — CHANGES_REQUESTED** [review]

**Review by hlissner — CHANGES_REQUESTED** [review]
Assuming `s/setq/setq!` works, please squash your two commits and reword it to conform to [our git conventions](https://gist.github.com/hlissner/4d78e396acb897d9b2d8be07a103a854):

```
fix(default): use persp-keymap-prefix's setter
```

**Comment by shackra** [comment]
> Assuming `s/setq/setq!` works, please squash your two commits and reword it to conform to [our git conventions](https://gist.github.com/hlissner/4d78e396acb897d9b2d8be07a103a854):
> 
> ```
> fix(default): use persp-keymap-prefix's setter
> ```
> 
> > I know this is a small change, but I wonder if something is missing that could get this approved and merge
> 
> Sorry for the late response. I've been ill the past two months, so my activity's been spotty. I am only returning to Doom now.

yeah, `setq!` works fine.

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
Thanks for your help!
